plugins/trayicon.py

```diff
@@ -174,10 +175,11 @@ def save_config(self, minimize_setting):
             f.write(minimize_setting)
 
     def trayicon_click(self, widget, data = None):
-        self.window.deiconify()
```

Comment:
I fear we need to keep the deiconify() also. This is for window managers that still do icon minimization (think OpenStep, Enlightement) where you iconify Liferea and want it deiconified when clicking on the tray.

Reply:
I think that should be covered here: https://github.com/lwindolf/liferea/pull/826/commits/a1dda20aa7dacb429aa931d9b67cb475805bc3ab#diff-7386da2621ccaf16c69c72a556b8263eR1384.

Reply:
Oh... I was to focussed on the trayicon code!